DokuwikiXMLExport.php

```diff
@@ -129,7 +129,7 @@ public function generateXMLExport($start, $count)
 
         $items = array();
         foreach ($this->pages as $key => $page) {
-            $item = $exporter->createItem($key);
+            $item = $exporter->createItem($start + $key);
```

Comment:
Why did you change that? If it was a bug write a test for it.

Reply:
Totally agreed, forgot about it.